Client/iTeamTalk/iTeamTalk/ChannelListViewController.swift

```diff
@@ -793,6 +793,7 @@ class ChannelListViewController :
 
         case CLIENTEVENT_CON_LOST :
             
+            playSound(.srv_LOST)
```

Comment:
This is already played here: https://github.com/BearWare/TeamTalk5/blob/fa14b4c1fa46386f5d9ef5fa774acc637a1ba9f7/Client/iTeamTalk/iTeamTalk/MainTabBarController.swift#L320